packages/Telephony/Schemas/Microsoft.Telephony.TerminationCharacterBatchInput.schema

```diff
@@ -0,0 +1,39 @@
+﻿{
+  "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+  "$role": "implements(Microsoft.IDialog)",
+  "title": "Termination Character Batch Input",
+  "description": "Prompts the user for multiple inputs that are aggregated until the termination string is received.",
+  "type": "object",
+  "required": [
+    "terminationCharacter",
+    "prompt",
+    "*"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "terminationCharacter": {
+      "$ref": "schema:#/definitions/stringExpression",
+      "title": "Termination Character",
+      "description": "Checks the latest character against the string and returns the completed batch if a match is found. Result will include specified character.",
```

Comment:
The current action allows terminationCharacter to be specified as more than one character - ex: "123#"